src/aurelia-api.js

```diff
@@ -4,4 +4,6 @@ export function configure(aurelia, configCallback) {
   let config = aurelia.container.get(Config);
 
   configCallback(config);
+
+  config.configured = true;
```

Comment:
i guess you figured it out.

the reason is, that changing to parsed response can mess up plugins which expect unparsed error. so i wanted to add a simple check that one doesn't change to parseError=true globally eg within a plugin